src/main/java/com/yo1000/sonarqube/collaboration/Application.java

```diff
@@ -5,6 +5,14 @@
  */
 public class Application {
     public static void main(String[] args) {
+        if (true) {
+            if (false) {
+                int a = 1;
+                return;
+            }
+            // TODO: Do nothing
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS1135)